Script/script.js

```diff
@@ -0,0 +1,4 @@
+function scrollToElement(targetId) {
+    var element = document.getElementById(targetId);
```

Comment:
Do not use `var` keyword.

+ good to know
ESLint `prefer-const` rule (https://eslint.org/docs/latest/rules/prefer-const): 
> If a variable is never reassigned, using the const declaration is better.
> const declaration tells readers, “this variable is never reassigned,” 
> reducing cognitive load and improving maintainability.